statscache/app.py

```diff
@@ -36,24 +28,36 @@ def main(name):
     )
 
 
+@app.route('/')
+def index():
+    """ Generate a JSON-P response with an index of plugins (as an array) """
+    return jsonp(json.dumps(plugins.keys()), 200)
```

Comment:
:+1: that sounds good to me.  (That's how we did it for bodhi2 and I quite like it).  We could expose RSS for people who want to be notified when new plugins become available.